rclcpp/include/rclcpp/node.hpp

```diff
@@ -96,6 +96,7 @@ class Node : public std::enable_shared_from_this<Node>
     const std::string & namespace_,
     rclcpp::Context::SharedPtr context,
     const std::vector<std::string> & arguments,
+    const std::vector<Parameter> & initial_values,
```

Comment:
In my opinion, as a user, `initial_values` is not descriptive enough; I think we're relying on the type to convey extra information. When I first looked at this PR I thought "initial values for what?" (and then as you saw, I wondered why it wasn't just called "initial parameters", especially since it sets the names as well). Were there other names that you considered?

Reply:
> Were there other names that you considered?
There were not

changed to `initial_paramters` plus added missing doxygen `\param[in] initial_parameters` in 29046af5d29a085c4edb81a698bdac9124978823